internal/db/namespaced.go

```diff
@@ -29,6 +30,15 @@ func NewNamespacedKV(db *leveldb.DB, prefix string) *NamespacedKV {
 	}
 }
 
+// Reset removes all entries in this namespace.
```

Comment:
This commit was not merged into master at the time.  I said in the initial PR that this depends on #1803.